### PR TITLE
Update README.md re: Netlify CLI

### PR DESCRIPTION
**Pull request by ttntm**
Added a note for Netlify CLI users; functions containing admin methods need to be deployed in order to work as intended.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -429,6 +429,8 @@ user
 
 The following admin methods are currently not available to be used directly. You can access `context.clientContext.identity` and get a short lived admin token through a Lambda function and achieve the same goals, e.g., update user role, create or delete user etc. See [Functions and Identity](https://docs.netlify.com/functions/functions-and-identity/) for more info.
 
+For users of [Netlify CLI](https://github.com/netlify/cli): please note, that functions using admin methods will _not_ work locally - they need to be deployed to your site in order to work as intended.
```
Comment (erezrokah) [inline README.md:432]:
```suggestion
> For users of [Netlify CLI](https://github.com/netlify/cli) - functions using admin methods will _not_ work locally - they need to be deployed to your site in order to work as intended.
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @ttntm, great addition. I suggested a small change, please let me know what you think.

**Comment by ttntm** [comment]
Thanks for the improved version, it's better that way.

**Review by erezrokah — APPROVED** [review]

**Pull request merged**